keyboards/cannonkeys/practice60/readme.md

```diff
@@ -10,3 +10,13 @@ Make example for this keyboard (after setting up your build environment):
     make cannonkeys/practice60:default
 
 See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+Build guide available at: https://docs.cannonkeys.com/bluepill-build/
+Blue Pill flashing guide at: https://docs.cannonkeys.com/flashing/
+Flash using dfu-util:
+
+    dfu-util.exe -d 1eaf:0003 -a 2 -D "\path\to\firmware.bin"
+
+Note: If using MSYS2 and getting AVR-GCC errors try running the command below and restarting MSYS2. See this link for more information: https://github.com/qmk/qmk_firmware/issues/4318
```

Comment:
Firmware for Arm chips isn't compiled with avr-gcc. I'm not sure what this has to do with this board.

Reply:
Apologies, I copy/pasted the wrong thing.  That link was what led me to the script to fix the errors though.